lib/http_adapter/angular.js

```diff
@@ -55,8 +55,16 @@ AngularAdapter.prototype.request = function(method, path, options, customHeaders
             headers[headerKeys[i]] = results[i];
         }
 
+        // Add cache buster to GETs -- we manage the cache ourselves.
+        var upperCaseMethod = method.toUpperCase();
+        if (upperCaseMethod === 'GET') {
+            path += (-1 === path.indexOf('?') ? '?' : '&');
```

Comment:
The problem is that the browser was caching these `GET` requests because they contain cache-control headers. The monocle client manages its own cache, so we don't want the browser to cache it. That's why we are adding a cache buster to `GET` requests.